### ISSUE TITLE
Shows Send fail after few times sending CAN FD packets.

### ISSUE THREAD

**Issue by dharmik768** [Longan-Labs/Longan_Labs_CAN_FD_Shield_Arduino#1]
I am using CAN  FD shield with Arduino board.
The problem is that it sends CAN FD packets the first few times like 2-3 times and I can see the TX led on shield blinking so its sending packets and in the serial monitor, it shows send ok message but after the first few tries it shows send fail continuously no matter how many times I send packets it shows send fail to me.
Is it like I am making some mistake in code or I have to call something each time after send ok like that?
Thank you for your help,
Dharmik.
[test_code.zip](https://github.com/Longan-Labs/Longan_Labs_CAN_FD_Shield_Arduino/files/6000210/test_code.zip)

**Comment by jakka351** [comment]
Yeah see how they dont respond ITS BECAUSE THEIR PRODUCTS DONT FUCKING ACTUALLY WORK
